<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Omar Mendoza Montoya</author>
 <class>qt_tools::plots::PlotWidget</class>
 <widget class="QWidget" name="qt_tools::plots::PlotWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>94</width>
    <height>138</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="contents">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="mainTextLabel">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Text</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QLabel" name="infoTextLabel">
     <property name="text">
      <string>Info Text</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGraphicsView" name="plotView">
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="renderHints">
      <set>QPainter::Antialiasing|QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
     </property>
     <property name="viewportUpdateMode">
      <enum>QGraphicsView::FullViewportUpdate</enum>
     </property>
    </widget>
   </item>
  </layout>
  <action name="saveAction">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="clipboardAction">
   <property name="text">
    <string>To Clipboard</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>plotView</tabstop>
 </tabstops>
 <connections/>
</ui>
